api/organisations/invites/views.py

```diff
@@ -85,6 +91,12 @@ class InviteLinkViewSet(
     def get_queryset(self):
         organisation_pk = self.kwargs.get("organisation_pk")
         user = self.request.user
+        organisation = Organisation.objects.get(id=organisation_pk)
+        if organisation.over_plan_seats_limit(shift=1):
+            raise PermissionDenied(
+                "The seats limit has been reached in your organisation."
+            )
```

Comment:
Should reuse the logic from [here](https://github.com/Flagsmith/flagsmith/blob/main/api/organisations/invites/views.py#L131-L136)

Reply:
done